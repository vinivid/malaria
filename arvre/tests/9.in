i Yara
i Ze
i Eliane
i Yara
i Ze
i Nadia
i Renata
i Queila
