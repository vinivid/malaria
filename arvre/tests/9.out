digraph {
"0 Yara" ->"1 Eliane" 
"0 Yara" ->"2 Ze" 
"1 Eliane" ->"4 Nadia" 
"4 Nadia" ->"10 Renata" 
"10 Renata" ->"21 Queila" 
}

digraph {
"0 Queila" ->"1 Eliane" 
"0 Queila" ->"2 Yara" 
"1 Eliane" ->"4 Nadia" 
"2 Yara" ->"5 Renata" 
"2 Yara" ->"6 Ze" 
}

digraph {
"0 Renata" ->"1 Nadia" 
"0 Renata" ->"2 Yara" 
"1 Nadia" ->"3 Eliane" 
"1 Nadia" ->"4 Queila" 
"2 Yara" ->"6 Ze" 
}

